code/game/objects/items/weapons/defib.dm

```diff
@@ -235,13 +235,13 @@
 /obj/item/defibrillator/compact/combat
 	name = "combat defibrillator"
 	desc = "A belt-equipped blood-red defibrillator that can be rapidly deployed. Does not have the restrictions or safeties of conventional defibrillators and can revive through space suits."
-	combat = 1
-	safety = 0
+	combat = TRUE
+	safety = FALSE
 
 /obj/item/defibrillator/compact/combat/loaded/New()
 	..()
 	paddles = make_paddles()
-	cell = new /obj/item/stock_parts/cell/infinite(src)
+	cell = new(src)
```

Comment:
This appears to be the functional change in this PR

Reply:
Yes.
> Exchanges the infinite cell in the Nukie Combat Defib with the regular cell that all defibs have (10k maxcharge)